### PR TITLE
Modification of InstructorStudentRecords before JSTL migration

### PR DESCRIPTION
**Pull request by wkurniawan07**
And yet another testimonial on how unstandardized things are...
I can guarantee this is the last standardization needed for this slice because I've managed to do the migration (the ones that reuse existing stuffs, at least) on top of this branch.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@taniach I hope you don't mind!

File: src/main/webapp/jsp/instructorStudentRecords.jsp [hunk 1]
```diff
@@ -105,7 +85,7 @@
                                         </tr>
                                         <tr>
                                             <td class="text-bold">
-                                                Email
+                                                Personal Email
                                             </td>
```
Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: Good you changed this. It was misleading earlier.

Comment (wkurniawan07) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: It was for standardizing purpose. Good that it has another benefit :p

File: src/main/webapp/jsp/instructorStudentRecords.jsp [hunk 2]
```diff
@@ -186,7 +184,7 @@
                                                     <div class="form-group form-inline">
                                                         <div class="form-group text-muted">
                                                             <p>
-                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name) %>:
+                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name + " (" + data.student.team + ", " + data.student.email + ")") %>:
                                                             </p>
```
Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: The email is already displayed after To. To Name (Team, Email)
Why display here again?

Comment (wkurniawan07) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: ...because it's like that in the other page that uses comment?

Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: In Instructor Comments, it is only 'To Name (Team, Email)'. There is no 'Comment about ...'. The email is displayed once. Having it twice in records looks like duplication.

Comment (wkurniawan07) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: I'm sorry to disagree:
image

Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: I see. That is after clicking the edit button. Before clicking 'edit' it is like this:
screen shot 2015-07-16 at 10 50 35 am
In both cases, the email is showed once.

I guess we have to choose between consistency and duplicate display in this case.

Comment (taniach) [inline src/main/webapp/jsp/instructorStudentRecords.jsp]: Since you are doing standardisation, keep the email for 'comment about' for now.

**Pull request merged**